### PR TITLE
Add authorization page for admins

### LINKED ISSUE #38: Add a authorization page for admins
add a place where admins can change non-admin users to admins and vice-versa

### PR DESCRIPTION
**Pull request by Izabellyrb**
resolves  #38

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
 
   // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
   // "remoteUser": "root",
+     "remoteUser": "vscode",
```
Comment (philipeleandro) [inline .devcontainer/devcontainer.json:29]: I think you can exclude this file in this PR. In my computer, it's all ok.

File: app/controllers/admin/application_controller.rb [hunk 2]
```diff
@@ -3,7 +3,6 @@
 module Admin
   class ApplicationController < Administrate::ApplicationController
     before_action :authenticate_user!
-    before_action :authenticate_admin
```
Comment (philipeleandro) [inline app/controllers/admin/application_controller.rb:6]: We can keep authenticate_admin in application controller

File: app/views/admin/welcomes/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+      <%= link_to t('users.index.title'), users_path, class: 'button login' %>
+    <% elsif user_signed_in? and !current_user.admin? %>
+      <%= link_to t('helpers.labels.edit_profile'), edit_user_registration_path, class: 'button login' %>
+      <%= button_to t('helpers.labels.logout'), destroy_user_session_path, method: :delete, class: 'button logout' %>
+    <% else %>
+      <%= link_to t('helpers.labels.sign_up'), new_user_registration_path, class: 'button login' %>
+      <%= link_to t('helpers.labels.login'), new_user_session_path, class: 'button login' %>
+    <% end %>
+  </div>
 </div>
```
Comment (philipeleandro) [inline app/views/admin/welcomes/index.html.erb:15]: add empty line

File: app/views/users/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          <td scope="row"><%= user.email %></td>
+          <td scope="row"><%= user.admin? ? t("users.index.is_admin") : t("users.index.is_user") %></td>
+          <td scope="row"><%= button_to t('helpers.change_admin_status'), change_admin_status_user_path(user), 
+                                        data: { turbo_confirm: t("users.messages.are_you_sure") }, method: :patch %></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+<%= link_to "Voltar", root_path , class: "button" %>
```
Comment (philipeleandro) [inline app/views/users/index.html.erb:31]: add empty line

Comment (Izabellyrb) [reply app/views/users/index.html.erb]: Added.

File: app/views/admin/welcomes/index.html.erb (hunk 3 above)
Comment (Izabellyrb) [reply app/views/admin/welcomes/index.html.erb]: Added.

**Pull request closed without merging**